src/components/base/accordion/Accordion.js

```diff
@@ -112,23 +111,17 @@ export function Item({
             additionalTxt && styles.firstelement,
           ].join(" ")}
         >
-          <Text
-            type="text2"
-            skeleton={isLoading}
-            lines="1"
-            className={[
-              animations["h-color-blue"],
-              animations["h-border-bottom"],
-            ].join(" ")}
-          >
-            {title}
-          </Text>
-          {subTitle && <Text type="text4">{subTitle}</Text>}
+          <Link tag={"span"}>
+            <Text type="text2" skeleton={isLoading} lines="1" tag={"span"}>
+              {title}
+            </Text>
+          </Link>
+          {subTitle && <Text tag={"span"} type="text4"></Text>}
```

Comment:
Generelt skal der lige tænkes over tagningen: 
Du ændre til `<span>` for at få display: inline - men der kan være nogle eksempler det er bedre hvis det forbliver et p tag. 

span tags er ligesom divs noget der bliver ignoreret semantisk, og er bare brugt til at ligge css på del elementer osv. I dette eksempel er data "subTitle" som måske endda indikere et header tag?